ansible/vars/topo_t2.yml

```diff
@@ -0,0 +1,2325 @@
+      vlans:
+        - "1.31@63"
+      vm_offset: 63
+    ARISTA33T1:
+      vlans:
+        - "2.0@64"
+      vm_offset: 64
+    ARISTA34T1:
+      vlans:
+        - "2.1@65"
+      vm_offset: 65
+    ARISTA35T1:
+      vlans:
+        - "2.2@66"
+      vm_offset: 66
+    ARISTA36T1:
+      vlans:
+        - "2.3@67"
+      vm_offset: 67
+    ARISTA37T1:
+      vlans:
+        - "2.4@68"
+      vm_offset: 68
+    ARISTA38T1:
+      vlans:
+        - "2.5@69"
+      vm_offset: 69
+    ARISTA39T1:
+      vlans:
+        - "2.6@70"
+      vm_offset: 70
+    ARISTA40T1:
+      vlans:
+        - "2.7@71"
+      vm_offset: 71
+    ARISTA41T1:
+      vlans:
+        - "2.8@72"
+      vm_offset: 72
+    ARISTA42T1:
+      vlans:
+        - "2.9@73"
+      vm_offset: 73
+    ARISTA43T1:
+      vlans:
+        - "2.10@74"
+      vm_offset: 74
+    ARISTA44T1:
+      vlans:
+        - "2.11@75"
+      vm_offset: 75
+    ARISTA45T1:
+      vlans:
+        - "2.12@76"
+      vm_offset: 76
+    ARISTA46T1:
+      vlans:
+        - "2.13@77"
+      vm_offset: 77
+    ARISTA47T1:
+      vlans:
+        - "2.14@78"
+      vm_offset: 78
+    ARISTA48T1:
+      vlans:
+        - "2.15@79"
+      vm_offset: 79
+    ARISTA49T1:
+      vlans:
+        - "2.16@80"
+      vm_offset: 80
+    ARISTA50T1:
+      vlans:
+        - "2.17@81"
+      vm_offset: 81
+    ARISTA51T1:
+      vlans:
+        - "2.18@82"
+      vm_offset: 82
+    ARISTA52T1:
+      vlans:
+        - "2.19@83"
+      vm_offset: 83
+    ARISTA53T1:
+      vlans:
+        - "2.20@84"
+      vm_offset: 84
+    ARISTA54T1:
+      vlans:
+        - "2.21@85"
+      vm_offset: 85
+    ARISTA55T1:
+      vlans:
+        - "2.22@86"
+      vm_offset: 86
+    ARISTA56T1:
+      vlans:
+        - "2.23@87"
+      vm_offset: 87
+    ARISTA57T1:
+      vlans:
+        - "2.24@88"
+      vm_offset: 88
+    ARISTA58T1:
+      vlans:
+        - "2.25@89"
+      vm_offset: 89
+    ARISTA59T1:
+      vlans:
+        - "2.26@90"
+      vm_offset: 90
+    ARISTA60T1:
+      vlans:
+        - "2.27@91"
+      vm_offset: 91
+    ARISTA61T1:
+      vlans:
+        - "2.28@92"
+      vm_offset: 92
+    ARISTA62T1:
+      vlans:
+        - "2.29@93"
+      vm_offset: 93
+    ARISTA63T1:
+      vlans:
+        - "2.30@94"
+      vm_offset: 94
+    ARISTA64T1:
+      vlans:
+        - "2.31@95"
+      vm_offset: 95
+
+configuration_properties:
+  common:
+    dut_asn: 65100
+    dut_type: LeafRouter
+    nhipv4: 10.10.246.254
+    nhipv6: FC0A::FF
+  spine:
+    swrole: spine
+  tor:
+    swrole: tor
+
+configuration:
+  ARISTA01T3:
+    properties:
+      - common
+      - spine
+    bgp:
+      asn: 65200
+      peers:
+        65100:
+          - 10.0.0.0
+          - FC00:1
+    interfaces:
+      Loopback0:
+        ipv4: 100.1.0.1/32
+        ipv6: 2064:100::1/128
+      Ethernet1:
+        ipv4: 10.0.0.1/31
+        ipv6: FC00:2/126
+    bp_interface:
+      ipv4: 10.10.246.1
+      ipv6: fc0a::2/64
```

Comment:
Generally T2 and T3 are connected by PortChannels. Probably we can design the topology like the t1-lag topo?
* Use LAG between T2 and 8 T3 VMs
* Use single link between T2 and 16 T3 VMs.

Totally 24 T3 VMs. 

From T2 to T1, we can use similar design, 8 LAGs to 8 T1 VMs. 16 links to 16 T1 VMs. Totally 24 T1 VMs.

Benefits of the proposal:
* The required footprint would be smaller
* Both LAG and single link can be covered

Reply:
Modified the testbed diagram, the testbed readme and the topo-t2 yml file as per your inputs. Thanks again